babel.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
```

Comment:
Since we are already running tsc with es5 output, is it necessary to also run babel? Couldn't we just bundle up the generated files?

The issue you pointed at with vega-util could maybe be resolved by requiring es5 compiled filed from eslint in rollup. What do you think?

Reply:
I'm not sure how to get roll-up to pull the ES5 files if that's possible. I didn't know if all the dependencies had ES5 builds and how to pull the right files when building an ES5 bundle.